CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## Changelog
 
 ### Version 2.1.0
-* Add support for tagged measurements (#121)
+* Add support for tagged measurements (#121). **NOTE**: This version introduces breaking changes for legacy sources. Please contact support@librato.com to learn more.
```

Comment:
This is a bit misleading, as you can still use sources. Breaking changes are only applicable to librato-rails and librato-rack, which uses either tags or sources.

Can you reword this to reflect that?